Platform.md

```diff
@@ -92,12 +90,12 @@ This file contains instructions for installing dependencies on different platfor
 
 ### Ubuntu 14.04
   * Install CMake [CMake Installation](https://cmake.org/install/)
-  * To update to OpenSSL 1.0.2
+  * To update to OpenSSL 1.1.0
     
     * `sudo apt-get install make` (if not already installed)
-    * `wget https://www.openssl.org/source/openssl-1.0.2g.tar.gz` 
-    * `tar -xzvf openssl-1.0.2g.tar.gz` 
-    * `cd openssl-1.0.2g`
+    * `wget https://www.openssl.org/source/openssl-1.1.0g.tar.gz` 
```

Comment:
I can't imagine that 'g' is still correct here.

Reply:
It works both with the `g` and without the `g`, but since it works without I removed it 👍